src/kernel/include/shared/cos_types.h

```diff
@@ -187,6 +187,9 @@ typedef enum {
 
 #define CAPTBL_EXPAND_SZ 128
 
+#define COS_VIRT_MACH_COUNT        2
```

Comment:
I'm moving this macro to vkernel, and will have it as `VM_COUNT` as you initially suggested and also for consistency sake. Else everywhere within vkernel, I'll have to expand VM/vm.